cornflow/shared/const.py

```diff
@@ -33,3 +33,58 @@
 AIRFLOW_TO_STATE_MAP = dict(
     success=EXEC_STATE_CORRECT, running=EXEC_STATE_RUNNING, failed=EXEC_STATE_ERROR
 )
+
+AUTH_DB = 1
+AUTH_LDAP = 2
+
+GET_ACTION = 1
```

Comment:
you could define: `ALL_DEFAULT_ACTIONS`, `ALL_DEFAULT_ACTIONS`, etc. If it's useful.

Reply:
Will add it